sdk/identity/azure-identity/src/azure_pipelines_credential.cpp

```diff
@@ -103,6 +103,7 @@ Request AzurePipelinesCredential::CreateOidcRequestMessage() const
   Request request = Request(HttpMethod::Post, requestUrl);
   request.SetHeader("content-type", "application/json");
   request.SetHeader("authorization", "Bearer " + m_systemAccessToken);
+  request.SetHeader("X-TFS-FedAuthRedirect", "Suppress");
```

Comment:
What does this header do?

Reply:
Please see the email thread on the topic :)

It prevents the service from responding with a redirect HTTP status code (useful for automation).